tests/test_jenkins.py

```diff
@@ -507,6 +570,21 @@ def test_get_jobs(self):
 
         self.assertEqual(response, body)
 
+    @httpretty.activate
+    def test_get_jobs_auth_api_token(self):
```

Comment:
I think this should check that the token is sent.

Reply:
The token is sent inside a basic authentication, maybe an assertion can be added to check whether in the list of headers there is `Authorization` one (which is must be `Basic`)